app/app/settings/base.py

```diff
@@ -100,19 +100,22 @@
 
 AUTH_PASSWORD_VALIDATORS = [
     {
-        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+        "NAME": "recordtransfer.validators.LengthRangeValidator",
+        "OPTIONS": {"min_length": 10, "max_length": 30},
     },
     {
-        "NAME": "django.contrib.auth.password_validation.MinimumLengthValidator",
-        "OPTIONS": {
-            "min_length": 8,
-        },
+        "NAME": "recordtransfer.validators.CharacterCategoriesValidator",
+        "OPTIONS": {"required_categories": 3},
     },
     {
-        "NAME": "django.contrib.auth.password_validation.CommonPasswordValidator",
+        "NAME": "recordtransfer.validators.PasswordHistoryValidator",
+        "OPTIONS": {"history_depth": 5},
     },
     {
-        "NAME": "django.contrib.auth.password_validation.NumericPasswordValidator",
+        "NAME": "recordtransfer.validators.UserAttributeContainsValidator",
+        "OPTIONS": {
+            "user_attributes": ("username", "first_name", "last_name"),
+        },
```

Comment:
Remove these options here, we'll disable overriding the default attributes since the validator only considers those three atrributes.


```suggestion
```